source/Halibut/Diagnostics/HalibutTimeoutsAndLimits.cs

```diff
@@ -45,14 +45,21 @@ public class HalibutTimeoutsAndLimits
         public int RewindableBufferStreamSize { get; set; } = 8192;
 
         /// <summary>
-        ///     Amount of time to wait for a TCP or SslStream read/write to complete successfully
+        /// Amount of time to wait for a TCP read/write to complete successfully.
+        ///
+        /// This Timeout is used when no other more specific timeout applies.
+        /// 
+        /// This applies to:
+        /// - Initial authentication and identification
```

Comment:
Shouldn't we use the lower timeout for - Initial authentication and identification